api-inference-community/docker_images/fairseq/app/pipelines/text_to_speech.py

```diff
@@ -39,6 +42,8 @@ def __call__(self, inputs: str) -> Tuple[np.array, int]:
         """
         tokenized_inputs = self._tokenize(inputs)
 
+        if not tokenized_inputs:
+            return np.zeros((0,)), self.sampling_rate
```

Comment:
If the inputs is `''` then there's a crash later. We can simply return nil in that case